src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -0,0 +1,53 @@
+package se.kth.sda.skeleton.comments;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import se.kth.sda.skeleton.posts.Post;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @NotBlank
+    private String body;
+
+    @ManyToOne
+    @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+    @JsonIdentityReference(alwaysAsId = true)
+    @JoinColumn(nullable = false)
+    @NotNull
+    private Post post;
```

Comment:
So, we will need to add corresponding @OneToMany on Post class

Reply:
We already did...so you don't do it as it's out of your scope, but probably we'll have some merge issues once we merge your or...